test/images/image-util.sh

```diff
@@ -92,8 +92,17 @@ build() {
   done
 }
 
+docker_version_check() {
+  docker_version=$(docker version --format '{{.Client.Version}}' | cut -d"-" -f1)
+  if [[ $docker_version != 18.06.0 && $docker_version < 18.06.0 ]]; then
```

Comment:
is this a string-wise comparison? will it do the right thing when 18.09.00 or whatever is released?

(also, minor style nits: prefer quoting variables and using `${var}` over `$var`. also don't need the semicolons ending the next two lines.)

Reply:
@ixdy yes it is string comparison, I tested with all possible combination, things are fine. here is the sample run result - https://gist.github.com/mkumatag/4003fc248e99a7a190a9e3a15a54a6a5